src/applications/discharge-wizard/components/v2/ReviewPage.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import { Link } from 'react-router';
+
+import { answerReviewLabel } from '../../helpers';
+import { SHORT_NAME_MAP } from '../../constants/question-data-map';
+import { pushToRoute } from '../../utilities/shared';
+import { pageSetup } from '../../utilities/page-setup';
+import { ROUTES } from '../../constants';
+
+const ReviewPage = ({ formResponses, router, viewedIntroPage }) => {
+  const H1 = 'Review your answers';
+  useEffect(
+    () => {
+      pageSetup(H1);
+    },
+    [H1],
+  );
+
+  useEffect(
+    () => {
+      if (!viewedIntroPage) {
+        router.push(ROUTES.HOME);
+      }
+    },
+    [router, viewedIntroPage],
+  );
+  const renderReviewAnswers = () => {
```

Comment:
Nit: consider legibility in your code with regard to empty lines between code blocks. This `useEffect` block followed immediately by a `const` definition makes it harder for my brain to digest (it may just be me).